weblate/utils/requirements.py

```diff
@@ -169,10 +167,75 @@ def get_versions():
     return result
 
 
+def get_version_number_db(database, version):
+    """
+        Returns version of database in format as (Major Version.Minor Version.Revision)
+        from an integer. To know more about database version , refer <https://www.psycopg.org/docs/connection.html>
+    """
+    if database is "POSTGRESQL":
+        major_version = version[0:2]
```

Comment:
This will break on postgres 9.x, the version there is for example "90209".

Reply:
I have fixed it.